net/freeradius/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=            freeradius
+PLUGIN_VERSION=         0.3.0
```

Comment:
just FYI: as long as the pull request is open you don't have to necessarily bump the version more than once. we also have PLUGIN_REVISION= which can be set to a numeric value for bugfixes, that's why most plugins only have X.Y, not X.Y.Z. But @fabianfrz likes three version numbers and that's perfectly fine here... only want to explain :)